### PR TITLE
Build instructions update

### PR DESCRIPTION
**Pull request by annabrewer**
Updated Windows build instructions to reflect the fact that the C++ tools for Visual Studio are still needed, and changed VS link to the one that actually includes VS 2017. 

Also updated the Android instructions with the above information, since downloading VS 2017 and the C++ tools was what fixed my build issues. Made a few other changes too: added instructions to configure a keystore for the app, updated the API levels, added the CMake version, removed some outdated Gradle stuff, added instructions for creating a build config, and wrote up a section on troubleshooting.

Lastly, merged BUILD_QUEST.md and BUILD_ANDROID.md, since they were nearly identical.

(Ignore the first commit below - I changed those things back)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

File: BUILD_ANDROID.md [hunk 1]
```diff
@@ -1,64 +1,104 @@
...
 * From the _Build_ menu select _Make Project_
-* Once the build completes, from the _Run_ menu select _Run App_ 
 
+### Running a Module
+
+* In the toolbar at the top of Android Studio, next to the green hammer icon, you should see a dropdown menu. From this menu, select _Edit Configurations_
+* Click the plus sign in the upper left corner of the new window, and select _Android App_
+* From the _Module_ drop down menu, select the module you are intending to run (most likely `interface` or `questInterface`)
+* From the _Launch_ drop down menu, select _Specified Activity_
+* In the _Activity_ field directly below, put `io.highfidelity.hifiinterface.PermissionChecker`
```
Comment (amerhifi) [inline BUILD_ANDROID.md]: io.highfidelity.<module Name>.PermisisonsChecker   

Where module name is module you are running. Select the one that applies.

File: BUILD_WIN.md [hunk 2]
```diff
@@ -7,10 +7,9 @@ Note: The prerequisites will require about 10 GB of space on your drive. You wil
...
 ### Step 1. Visual Studio 2017 & Python
 
-If you don’t have Community or Professional edition of Visual Studio 2017, download [Visual Studio Community 2017](https://www.visualstudio.com/downloads/).
+If you don’t have Community or Professional edition of Visual Studio 2017, download [Visual Studio Community 2017](https://visualstudio.microsoft.com/vs/older-downloads/).
 
-Leave default components.  
-If you do not already have a python development environment installed, also check  "Python Development" in this screen.
+When selecting components, check "Desktop development with C++". Also on the right on the Summary toolbar, check "Windows 8.1 SDK and UCRT SDK" and "VC++ 2015.3 v140 toolset (x86,x64)". If you do not already have a python development environment installed, also check "Python Development" in this screen.
 
 If you already have Visual Studio installed and need to add python, open the "Add or remove programs" control panel and find the "Microsoft Visual Studio Installer".  Select it and click "Modify".  In the installer, select "Modify" again, then check "Python Development" and allow the installer to apply the changes.
```
Comment (amerhifi) [inline BUILD_WIN.md]: Just download python 3.7 (don't use the Visual studio installer)

**Pull request closed without merging**